app/Views/errors/html/error_exception.php

```diff
@@ -178,7 +178,7 @@
 							<tr>
 								<td><?= htmlspecialchars($key, ENT_IGNORE, 'UTF-8') ?></td>
 								<td>
-									<?php if (! is_array($value) && ! is_object($value)) : ?>
+									<?php if (! is_array($value) && ! is_object($value) && ! is_resource($value)) : ?>
 										<?= htmlspecialchars($value, ENT_SUBSTITUTE, 'UTF-8') ?>
 									<?php else: ?>
 										<?= '<pre>' . print_r($value, true) ?>
```

Comment:
Though unrelated, I am just wondering. Should this line have a closing `</pre>` tag? Or it is unnecessary?

Also, there's another `htmlspecialchars` of `$value` at the bottom. Might as well include that.

Reply:
`</pre>` should be.